packages/wordpress-kit/README.md

```diff
@@ -28,10 +30,31 @@ const client = new GraphqlClientFactory(
 ).create();
 ```
 
+### TailwindcssPlugin
+
+1. Import the plugin to your `tailwind.config.js` file
+
+```
+const { tailwindcssPlugin } = require("@pantheon-systems/wordpress-kit");
+```
+
+2. Add the plugin to your `plugins` array:
+
+```
+module.exports = {
+  content: [ ... ],
+  theme: { ... },
+  plugins: [tailwindcssPlugin],
+};
+```
+
+3. (Optional) Extend the plugin or override the theme in the `theme.extend`
+   section of your tailwindcss config.
+
```

Comment:
Another optional step can be add 
`/** @type {import('@pantheon-systems/wordpress-kit').TailwindcssConfig} */
` above the module.exports in tailwind config to add autocomplete and types docs